### ISSUE TITLE
vue 3 support

### ISSUE THREAD

**Issue by softrebel** [dealloc/vuec#11]
Hi, 
I like to develop this package to add more features & supporting vue 3.
If you are interested please contact me.
Thanks

**Comment by dealloc** [comment]
I'm interested in seeing how you'd add support for Vue 3.
Would you inject the dependencies through the `setup` method?

**Comment by softrebel** [comment]
Yes, but first i'm gonna find a way to support ioc methods on setup or Vue instance and create injection way on it.

**Comment by dealloc** [comment]
I'm looking forward to seeing a PR from you
